docs/_docs/localization/index.md

```diff
@@ -40,25 +46,28 @@ Below are the steps to build the plugin. You can also find instructions at: <htt
    ```
 
 3. Install the figma desktop app. Figma will need to read the plugin code saved as a local file. You
-   can download it from the [Figma downloads page][2].
+   can download it from the [Figma downloads page][3].
 
 4. Import the plugin from manifest and you are ready to run the plugin. The file path is:
 
    ```text
    automotive-design-compose/support-figma/extended-layout-plugin/manifest.json
    ```
 
-   <img src="./localization/ImportPluginFromManifest.png">
+   <img src="./ImportPluginFromManifest.png">
 
 5. Run the plugin under development through:
-   **Figma Main Menu -> Plugins -> Development**
-   **-> Automotive Design For Compose -> Localization Tool -> Generate String Resource**
+   **Figma Main Menu -> Plugins -> Development -> Automotive Design For Compose -> Localization Tool**
 
-### 1.2 Strings generation options
+## 2. String resource tool
+
+### 2.1 Generate string resources
+
+#### 2.1.1 Strings generation options
 
 Currently the plugin provides three options to exclude text nodes from translations.
 
-<img src="./localization/LocalizationOptions.png">
+<img src="./LocalizationOptions.png">
```

Comment:
I remember an issue with the public site when these images were not in the proper folder, do you know if that's still the case?

Reply:
I moved the md file under the folder. That is why I don't need this any more.